startlxqtwayland.in

```diff
@@ -73,6 +73,10 @@ if grep -q "compositor" "$XDG_CONFIG_HOME/lxqt/session.conf"; then
     COMPOSITOR=`cat "$XDG_CONFIG_HOME"/lxqt/session.conf|grep compositor |awk -F'=' '{sub(".*/", "", $2); print $2}'`
 fi
 
+if [ -z "$COMPOSITOR" ] && [ -f "/usr/share/lxqt/wayland/default-compositor" ]; then
```

Comment:
We definitely shouldn't use hardcoded path. The XDG_CONFIG_DIRS dirs should be considered here, IMO.

Reply:
For a distribution default in the event the user hasn't picked anything, user config directories are inappropriate.